src/components/positions-table/helpers/cancelOrders.tsx

```diff
@@ -58,6 +58,7 @@ export async function cancelOrders(props: CancelOrdersPropsI) {
                       ]}
                     />
                   );
+                  waitForTransactionReceipt(tradingClient, { hash: tx.hash, timeout: 30_000 }).then();
```

Comment:
I think , await should be used here.
But, i'm not sure it will work as expected.

```suggestion
                  await waitForTransactionReceipt(tradingClient, { hash: tx.hash, timeout: 30_000 });
```